tests/appsec/test_asm_standalone.py

```diff
@@ -741,6 +741,30 @@ def test_app_dependencies_loaded(self):
                 raise Exception(dependency + " not received in app-dependencies-loaded message")
 
 
+@rfc("https://docs.google.com/document/d/12NBx-nD-IoQEMiCRnJXneq4Be7cbtSc6pJLOFUWTpNE/edit")
+@features.appsec_standalone
+@scenarios.appsec_no_stats
+class Test_AppSecStandalone_NotEnabled:
+    """Test expected behaviour when standalone is not enabled."""
+
+    def setup_client_computed_stats_header_is_not_present(self):
+        trace_id = 1212121212121212122
+        self.r = weblog.get(
+            "/",
+            headers={
+                "x-datadog-trace-id": str(trace_id),
+            },
+        )
```

Comment:
I believe you have to provide the `x-datadog-parent-id` header along with `x-datadog-trace-id`.
If not `x-datadog-parent-id` is ignored (at least this is how [nodejs tracer works](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/opentracing/propagation/text_map.js#L502))

```suggestion
        trace_id = 1212121212121212122
        parent_id = 34343434
        self.r = weblog.get(
            "/",
            headers={
                "x-datadog-trace-id": str(trace_id),
                "x-datadog-parent-id": str(parent_id),
            },
        )
```

Reply:
Thanks!